### PR TITLE
Add new content field: firstPublishedDate.

### PR DESCRIPTION
**Pull request by iulianp26**

### CODE REVIEW

File: video_mapper.go [hunk 1]
```diff
@@ -326,7 +331,7 @@ func unsafeJSONMarshal(v interface{}) ([]byte, error) {
 	return b, nil
 }
 
-func isPublishEvent(video map[string]interface{}) (publishEvent bool, err error) {
+func isPublishEvent(video map[string]interface{}) (_ bool, err error) {
```
Comment (BerniVarga) [inline video_mapper.go]: I don't think it has an advantage using _ for named return type here. 
Maybe it's even simpler if we don't use named return types here at all, cause we don't benefit neither from the default init it offers, neither from the return clause.

File: video_mapper.go [hunk 2]
```diff
@@ -420,7 +425,16 @@ func getByline(brightcoveVideo map[string]interface{}, publishReference string)
...
 }
 
-func get(key string, brightcoveVideo map[string]interface{}) (val string, err error) {
+func getFirstPublishedDate(video map[string]interface{}) (result string) {
+	result, err := get("published_at", video)
+	if err != nil {
+		warnLogger.Printf("No valid value could be found for firstPublishedDate: [%v]", err)
+	}
+
+	return result
```
Comment (BerniVarga) [inline video_mapper.go:435]: In short functions with named results types, you could simply use a return.

File: video_mapper.go [hunk 3]
```diff
@@ -420,7 +425,16 @@ func getByline(brightcoveVideo map[string]interface{}, publishReference string)
 	return byline
 }
 
-func get(key string, brightcoveVideo map[string]interface{}) (val string, err error) {
+func getFirstPublishedDate(video map[string]interface{}) (result string) {
+	result, err := get("published_at", video)
```
Comment (BerniVarga) [inline video_mapper.go]: Would it make sense to extract the field name into a variable? Although, I don't have strong views on this... Also, is it intentional to use the same field for publishDate and firstPublishDate?

File: video_mapper.go (hunk 1 above)
Comment (iulianp26) [reply video_mapper.go]: I have added _ in first place due to IDEA compilation warning. Fixed this by removing usage of name return values.

File: video_mapper.go (hunk 3 above)
Comment (iulianp26) [reply video_mapper.go]: Add it as constant. The requirement is that for video we should use value from published_at for first publication date.

File: video_mapper.go (hunk 2 above)
Comment (iulianp26) [reply video_mapper.go:435]: Removed named return value.

**Review by ioanaciont — APPROVED** [review]

**Pull request merged**